.travis.yml

```diff
@@ -17,3 +24,4 @@ script:
 
 after_script:
     - docker-compose -f docker-compose.yaml down
+    - .docker/push-image.sh
```

Comment:
Will push after failures too. Probably ok.

Reply:
That's _really_ unstable, will move to `after_success`